bin/review-rot

```diff
@@ -1,57 +1,42 @@
 #!/usr/bin/env python
 
 import argparse
+import collections
 import logging
 import operator
 import os
-from os.path import expanduser, expandvars
 import urllib
 
-from reviewrot import get_git_service, GerritService
+from os.path import expanduser, expandvars
+
+from shutil import copyfile
+
+from reviewrot import GerritService, get_git_service
 
 import yaml
 
+log = logging.getLogger('review-rot')
+
 
-def main(state_, value, duration, ssl_verify=None):
+def main():
     """
     Reads input configuration file.
     Calls appropriate git service with suitable inputs.
-
-    state_ (str): The filter(state) for review requests, e.g, older or newer
-    value (int): The value in terms of duration for requests
-                 to be older or newer than
-    duration (str): The duration in terms of period(year,month,hour,minute)
-                    for requests to be older or newer than.
     """
-    log = logging.getLogger('review-rot')
-    home = expanduser('~')
+    config = load_config_file()
 
-    path_to_config = os.path.join(home, '.reviewrot.yaml')
-    if not os.path.exists(path_to_config):
-        raise RuntimeError("No config '.reviewrot.yaml' provided. Please"
-                           " specify it in home directory: %s" % home)
-    # read input from home directory for pull requests
-    with open(path_to_config, 'r') as f:
-        config = yaml.load(f)
+    arguments = get_arguments(config.get('arguments'))
 
     results = []
-    for item in config:
+
+    for item in config.get('git_services'):
```

Comment:
If the user doesn't specify `git_services` in the config, this will result in a `NoneType` error. Perhaps defaulting to a list would be a nicer user experience.

```python
for item in config.get('git_services', []):
```